hera_pspec/pspecdata.py

```diff
@@ -681,6 +685,75 @@ def get_G(self, key1, key2):
                 G[i,j] += np.einsum('ab,ba', iR1Q[i], iR2Q[j])
 
         return G / 2.
+
+    def get_H(self, key1, key2, taper='none'):
+        """
+        Calculates the response matrix H of the unnormalized band powers q
+        to the true band powers p, i.e.,
+
+            <q_a> = \sum_b H_{ab} p_b
+
+        This is given by
+
+            H_ab = (1/2) Tr[R_1 Q_a^alt R_2 Q_b]
+
+        (See HERA memo #44). As currently implemented, this approximates the
+        primary beam as frequency independent. Under this approximation, the
+        our H_ab is defined using the equation above *except* we have
+        Q^{yet_another} rather than Q_b, where
```

Comment:
`yet_another` is not a great name for a variable...

Reply:
Changed it to `Q^{tapered}`